tests/Dfe.EarlyYearsQualification.UnitTests/Controllers/HomeControllerTests.cs

```diff
@@ -29,38 +30,41 @@ public async Task Index_ContentServiceReturnsNoContent_RedirectsToErrorPage()
         _mockContentService.Setup(x => x.GetStartPage()).ReturnsAsync((StartPage)default!);
         var result = await _controller!.Index();
 
-        Assert.IsNotNull(result);
+        result.Should().NotBeNull();
+
         var resultType = result as RedirectToActionResult;
-        Assert.IsNotNull(resultType);
-        Assert.AreEqual("Error", resultType.ActionName);
+        resultType.Should().NotBeNull();
+        resultType!.ActionName.Should().Be("Error");
     }
 
     [TestMethod]
     public async Task Index_ContentServiceReturnsContent_ReturnsStartPageModel()
     {
         var startPageResult = new StartPage
-        {
-            CtaButtonText = "Start now",
-            Header = "This is the header",
-            PostCtaButtonContentHtml = "This is the post cta content",
-            PreCtaButtonContentHtml = "This is the pre cta content",
-            RightHandSideContentHeader = "This is the side content header",
-            RightHandSideContentHtml = "This is the side content"
-        };
+                              {
+                                  CtaButtonText = "Start now",
+                                  Header = "This is the header",
+                                  PostCtaButtonContentHtml = "This is the post cta content",
+                                  PreCtaButtonContentHtml = "This is the pre cta content",
+                                  RightHandSideContentHeader = "This is the side content header",
+                                  RightHandSideContentHtml = "This is the side content"
+                              };
```

Comment:
Is the indentation correct here? Seems too far in or is this one of the resharper rules?

Reply:
This is what Rider did by default. If you get hold of Rider, we can then agree what our code format settings are, bake them in and document them somewhere, and then get ReSharper to implement them for us across the board. 🙂